docs/user/custom-resources/apirule/v1beta2/04-15-api-rule-access-strategies.md

```diff
@@ -0,0 +1,78 @@
+# APIRule v1beta2 Access Strategies
+
+APIRule allows you to define the security configuration for an exposed endpoint using the concept of access strategies. The supported handlers for APIRule `v1beta2` are `no_auth` and `jwt`.
```

Comment:
I think overall we should we should find a common name for noAuth, extAuth and jwt in the docs. We called it handler, because it was in a handler field in the old structure. I find it confusing that we use the terms access strategy and handler interchangeably.

Reply:
Yes, I agree
What about sticking to "access strategy"? We don't have this filed either, but I think we used it the most often in the docs

Reply:
Yes, I think we can use access strategy.